nautobot/core/tests/test_releases.py

```diff
@@ -69,9 +69,9 @@ class GetReleasesTestCase(SimpleTestCase):
     @patch.object(requests, "get")
```

Comment:
The cases in this file aren't examples, they're mocks. I think it would be clearer to call them `mock_cache_get`, `mock_cache_set`, `mock_request_get`.